SPECS/dwarves/dwarves.spec

```diff
@@ -0,0 +1,506 @@
+gcc, with the same compiler flags, and then use codiff to make sure the
+original .o file and the new one generated from debug info produces the same
+debug info.
+
+Pahole also can be used to use all this type information to pretty print raw data
+according to command line directions.
+
+Headers can have its data format described from debugging info and offsets from
+it can be used to further format a number of records.
+
+The btfdiff utility compares the output of pahole from BTF and DWARF to make
+sure they produce the same results.
+
+%package -n %{libname}%{libver}
+Summary:        Debugging information  processing library
+
+%description -n %{libname}%{libver}
+Debugging information processing library.
+
+%package -n %{libname}%{libver}-devel
+Summary:        Debugging information library development files
+Requires:       %{libname}%{libver} = %{version}-%{release}
+
+%description -n %{libname}%{libver}-devel
+Debugging information processing library development files.
+
+%prep
+%setup -q
+%patch1 -p1
+
+%build
+%cmake -DCMAKE_BUILD_TYPE=Release .
+%cmake_build
+
+%install
+rm -Rf %{buildroot}
+%cmake_install
+
+%ldconfig_scriptlets -n %{libname}%{libver}
+
+%files
+%doc README.ctracer
+%doc README.btf
+%doc changes-v1.21
+%doc NEWS
+%{_bindir}/btfdiff
+%{_bindir}/codiff
+%{_bindir}/ctracer
+%{_bindir}/dtagnames
+%{_bindir}/fullcircle
+%{_bindir}/pahole
+%{_bindir}/pdwtags
+%{_bindir}/pfunct
+%{_bindir}/pglobal
+%{_bindir}/prefcnt
+%{_bindir}/scncopy
+%{_bindir}/syscse
+%{_bindir}/ostra-cg
+%dir %{_datadir}/dwarves/
+%dir %{_datadir}/dwarves/runtime/
+%dir %{_datadir}/dwarves/runtime/python/
+%defattr(0644,root,root,0755)
+%{_mandir}/man1/pahole.1*
+%{_datadir}/dwarves/runtime/Makefile
+%{_datadir}/dwarves/runtime/linux.blacklist.cu
+%{_datadir}/dwarves/runtime/ctracer_relay.c
+%{_datadir}/dwarves/runtime/ctracer_relay.h
+%attr(0755,root,root) %{_datadir}/dwarves/runtime/python/ostra.py*
+
+%files -n %{libname}%{libver}
+%{_libdir}/%{libname}.so.*
+%{_libdir}/%{libname}_emit.so.*
+%{_libdir}/%{libname}_reorganize.so.*
+
+%files -n %{libname}%{libver}-devel
+%doc MANIFEST README
+%{_includedir}/dwarves/btf_encoder.h
+%{_includedir}/dwarves/config.h
+%{_includedir}/dwarves/ctf_encoder.h
+%{_includedir}/dwarves/ctf.h
+%{_includedir}/dwarves/dutil.h
+%{_includedir}/dwarves/dwarves.h
+%{_includedir}/dwarves/dwarves_emit.h
+%{_includedir}/dwarves/dwarves_reorganize.h
+%{_includedir}/dwarves/elfcreator.h
+%{_includedir}/dwarves/elf_symtab.h
+%{_includedir}/dwarves/gobuffer.h
+%{_includedir}/dwarves/hash.h
+%{_includedir}/dwarves/libbtf.h
+%{_includedir}/dwarves/libctf.h
+%{_includedir}/dwarves/list.h
+%{_includedir}/dwarves/rbtree.h
+%{_includedir}/dwarves/pahole_strings.h
+%{_libdir}/%{libname}.so
+%{_libdir}/%{libname}_emit.so
+%{_libdir}/%{libname}_reorganize.so
+
+%changelog
+* Fri Aug 20 2021 Chris Co <chrco@microsoft.com> - 1.21-4
+- Initial CBL-Mariner import from Fedora 35 (License: GPLv2)
```

Comment:
I believe we should be using the license of the spec file itself, rather than the license of the package here. Would you mind changing this to `(License: MIT)` to match that convention?